src/schema/resolvers.js

```diff
@@ -348,12 +346,14 @@ module.exports = {
 
       // Insert Transaction
       const tx = {
-        txid,
-        version,
+        txid: sentTx.txid,
         type,
         status: txState.PENDING,
         createdTime: moment().unix(),
+        gasLimit: sentTx.args.gasLimit,
```

Comment:
if `gasLimt` can be passed as a parameter from UI, shouldnt `gasLimit` can included in input in mutation and be read from `data`?

Reply:
the gasLimit is set from the API layer contract call. don't feel like the UI should be handling the calculations for setting the limit.